src/azure-cli/azure/cli/command_modules/security/custom.py

```diff
@@ -108,9 +112,22 @@ def list_security_settings(client):
     return client.list()
 
 
-def get_security_setting(client, resource_name):
+def get_security_setting(client, setting_name):
 
-    return client.get(resource_name)
+    return client.get(setting_name)
+
+
+def update_security_setting(client, setting_name, enabled):
+
+    if setting_name == Enum69.SENTINEL:
+        setting = AlertSyncSettings()
+    elif setting_name in (Enum69.MCAS, Enum69.WDATP, Enum69.WDATP_EXCLUDE_LINUX_PUBLIC_PREVIEW):
+        setting = DataExportSettings()
+    else:
+        setting = Setting()
```

Comment:
When you set `arg_type=get_enum_type(Enum69)` in _params.py, these two lines will never be run, because the setting_name can only be one of the Enum69's values.

Reply:
thank you